clingraph/__init__.py

```diff
@@ -1,188 +1,305 @@
-from clingo.application import Application, clingo_main, Flag
-from clingo.symbol import Function
+import imageio
+import os
```

Comment:
This is how the lint suggested

Reply:
Okay, that is weird because this is the output of `pylint clingraph` at my end. What am I doing different than you?

```
$ pylint clingraph/
************* Module clingraph
clingraph/__init__.py:89:31: C0303: Trailing whitespace (trailing-whitespace)
clingraph/__init__.py:95:31: C0303: Trailing whitespace (trailing-whitespace)
clingraph/__init__.py:96:29: C0303: Trailing whitespace (trailing-whitespace)
clingraph/__init__.py:100:42: C0303: Trailing whitespace (trailing-whitespace)
clingraph/__init__.py:108:52: C0303: Trailing whitespace (trailing-whitespace)
clingraph/__init__.py:120:78: C0303: Trailing whitespace (trailing-whitespace)
clingraph/__init__.py:85:20: W1401: Anomalous backslash in string: '\o'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
clingraph/__init__.py:1:0: C0114: Missing module docstring (missing-module-docstring)
clingraph/__init__.py:13:0: R0902: Too many instance attributes (13/7) (too-many-instance-attributes)
clingraph/__init__.py:37:4: W0237: Parameter 'control' has been renamed to 'ctl' in overridden 'Clingraph.main' method (arguments-renamed)
clingraph/__init__.py:52:31: W0622: Redefining built-in 'dir' (redefined-builtin)
clingraph/__init__.py:56:28: W0622: Redefining built-in 'format' (redefined-builtin)
clingraph/__init__.py:72:26: W0622: Redefining built-in 'type' (redefined-builtin)
clingraph/__init__.py:140:8: W0104: Statement seems to have no effect (pointless-statement)
clingraph/__init__.py:170:20: W0106: Expression "structure[e][s2id(atom.arguments[0], e)]" is assigned to nothing (expression-not-assigned)
clingraph/__init__.py:278:25: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
clingraph/__init__.py:304:0: C0116: Missing function or method docstring (missing-function-docstring)
clingraph/__init__.py:2:0: C0411: standard import "import os" should be placed before "import imageio" (wrong-import-order)
clingraph/__init__.py:3:0: C0411: standard import "import copy" should be placed before "import imageio" (wrong-import-order)
clingraph/__init__.py:5:0: C0411: standard import "from collections import defaultdict" should be placed before "import imageio" (wrong-import-order)
************* Module clingraph.__main__
clingraph/__main__.py:1:0: C0114: Missing module docstring (missing-module-docstring)

------------------------------------------------------------------
Your code has been rated at 8.76/10 (previous run: 8.58/10, +0.18)
```

Reply:
Different versions of pylint?

Reply:
I think just not properly installed in the right coda env.